doc/deployment/aws_cloudfront.md

```diff
@@ -0,0 +1,78 @@
+# AWS Cloudfront
+
+For a production installation we recommend using cloudfront.
+
+To deploy a production ready AWS cluster:
+
+1) Follow the instructions for a normal AWS deployment
```

Comment:
I mean that you could make the things you have labeled as 1), 2), etc. actual sections similar to the other deploy docs.  Because the 1), 2), etc. don't look that great in markdown.